src/Tribe/Views/V2/Widgets/List_Widget.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * List Widget
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ * @since   TBD
+ */
+
+namespace Tribe\Events\Views\V2\Widgets;
+
+use Tribe\Events\Views\V2\Manager as Views_Manager;
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\Views\Widgets\List_Widget_View;
+use Tribe\Widget\Widget_Abstract;
+use Tribe__Context as Context;
+use Tribe__Utils__Array as Arr;
+
+/**
+ * Class for List Widget.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+class List_Widget extends Widget_Abstract {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected $slug = 'tribe_events_list_widget';
```

Comment:
:no_entry_sign: Missing @var tag in member variable comment

<sub>hash: 68cbc5901f7e934333ca30811d37c52f</sub>